src/Native/Canvas.js

```diff
@@ -222,6 +222,41 @@ var _elm_community$canvas$Native_Canvas = function () {  // eslint-disable-line
       ctx.scale(drawOp._0, drawOp._1);
       break;
 
+    case "SetLineDash" :
+
+      ctx.setLineDash(_elm_lang$core$Native_List.toArray(drawOp._0)); // eslint-disable-line no-undef
```

Comment:
I'm starting to think that for elm core calls, we can toss something like this to the top of the file:

```
/*global _elm_lang$core$Native_List */
```

We can add other things in there like scheduler, or whatever else we use. Not sure if it's necessary, since I don't know off hand how many places we're doing this right now, and it's mostly my doing (oopsies). Thoughts?